### PR TITLE
Feature- Add handleChange() prop to ColorPicker

### PR DESCRIPTION
**Pull request by NeenaDev**
**Requires:** https://github.com/vanilla/internal/pull/2265

**Changes:**
Add Add handleChange() prop to `<ColorPicker />`  for updating assets redux side.

**Update:**  - Remove handleChange() prop to `<ColorPicker />`. Now updating assets on formik validate  

**Closes:** https://github.com/vanilla/knowledge/issues/1571

### CODE REVIEW

File: library/src/scripts/forms/themeEditor/ColorPicker.tsx [hunk 1]
```diff
@@ -56,15 +63,20 @@ export default function ColorPicker(props: IColorPicker) {
     const onTextChange = e => {
         const colorString = e.target.value;
         helpers.setTouched(true);
+
         if (stringIsValidColor(colorString)) {
             setValidColor(colorString); // Only set valid color if passes validation
+            errorHelpers.setValue(undefined);
```
Comment (slafleche) [inline library/src/scripts/forms/themeEditor/ColorPicker.tsx]: Aren't you missing a helpers.setValue(colorString) here?

File: library/src/scripts/forms/themeEditor/ColorPicker.tsx [hunk 2]
```diff
@@ -87,6 +100,8 @@ export default function ColorPicker(props: IColorPicker) {
     return (
         <>
             <span className={classes.root}>
+                <input className={visibility().displayNone} {...errorField} />
```
Comment (slafleche) [inline library/src/scripts/forms/themeEditor/ColorPicker.tsx]: What is this used for?

File: library/src/scripts/forms/themeEditor/themeEditorStyles.ts [hunk 3]
```diff
@@ -48,7 +62,7 @@ export const themeEditorClasses = useThemeCache(() => {
     const frame = style(
         "frame",
         {
-            width: calc(`${percent(vars.frame.width)} - ${unit(vars.panel.width)}`),
+            // width: calc(`${percent(vars.frame.width)} - ${unit(vars.panel.width)}`),
```
Comment (slafleche) [inline library/src/scripts/forms/themeEditor/themeEditorStyles.ts]: clean up comments

File: library/src/scripts/forms/themeEditor/themeEditorStyles.ts [hunk 4]
```diff
@@ -69,10 +83,23 @@ export const themeEditorClasses = useThemeCache(() => {
             width: percent(100),
         }),
     );
+    const frameContainer = style(
+        "frameContainer",
+        {
+            // width: calc(`${percent(vars.frameContainer.width)} - ${unit(vars.styleOptions.width)}`),
```
Comment (slafleche) [inline library/src/scripts/forms/themeEditor/themeEditorStyles.ts]: clean up comments

File: library/src/scripts/forms/themeEditor/ColorPicker.tsx (hunk 1 above)
Comment (NeenaDev) [reply library/src/scripts/forms/themeEditor/ColorPicker.tsx]: Fixed.

File: library/src/scripts/forms/themeEditor/themeEditorStyles.ts (hunk 3 above)
Comment (NeenaDev) [reply library/src/scripts/forms/themeEditor/themeEditorStyles.ts]: Removed commented lines

File: library/src/scripts/forms/themeEditor/themeEditorStyles.ts (hunk 4 above)
Comment (NeenaDev) [reply library/src/scripts/forms/themeEditor/themeEditorStyles.ts]: Fixed

**Pull request closed without merging**